mm2src/coins/eth.rs

```diff
@@ -249,6 +249,42 @@ impl From<web3::Error> for BalanceError {
     fn from(e: web3::Error) -> Self { BalanceError::Transport(e.to_string()) }
 }
 
+#[derive(Debug, Display, Serialize, SerializeErrorType)]
+#[serde(tag = "error_type", content = "error_data")]
+#[allow(dead_code)]
+pub enum EthActivationV2Error {
+    InvalidPayload(String),
+    #[display(fmt = "Platform coin {} activation failed. {}", ticker, error)]
+    ActivationFailed {
+        ticker: String,
+        error: String,
+    },
+    CouldNotFetchBalance(String),
+    UnreachableNodes(String),
+    AtLeastOneNodeRequired(String),
```

Comment:
Please remove String from this variant and use `#[display(fmt = "Enable request for ETH coin must have at least 1 node")]` instead.

Reply:
done